### ISSUE TITLE
Replace whitespace with underscores in benchmark plot/table version labels and file names

### ISSUE THREAD

**Issue by yantosca** [geoschem/gcpy#316]
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Confirm you have reviewed the following documentation

- [x] [Contributing guidelines](https://gcpy.readthedocs.io/en/stable/reference/CONTRIBUTING.html)

### New GCPy feature or discussion
We propose replacing spaces and tabs with "_" characters in benchmark plot and table file names and version strings.  Currently, it is possible to specify a version string with spaces, e.g.:
```yaml
data:
  ref:
    gcc:
      version: gcc 14.2.0 rc 0
      ...
    gchp:
      version: gchp 14.2.0 rc 0
  ...
  dev:
    gcc:
      version: gcc 14.2.0 rc 2
      ...
    gchp:
      version: gchp 14.2.0 rc 2
       ...
```
and this would result in several file names for benchmark plots and tables having spaces in them.

**Comment by yantosca** [comment]
NOTE: PR #319 contains a new function named `replace_whitespace` that will strip whitespace from labels and file names.  It has been applied to some of the newer benchmark scripts but needs to be applied to all benchmark scripts.

**Comment by yantosca** [comment]
This issue can now be closed as PR #340 has been merged into the GCPy 1.6.0 development stream.
